src/scss/components/_back-to-top.scss

```diff
@@ -0,0 +1,24 @@
+@use 'abstracts' as *;
+
+.back-to-top {
+  display: none;
+
+  @include breakpoint('large') {
+    position: absolute;
```

Comment:
After discussing with Tucker, this should actually be `position: fixed`, so it's always visible.